site/en/codelabs/openthread-border-router/index.lab.md

```diff
@@ -303,7 +201,7 @@ mDNS has been widely used for publishing DNS-SD service on link-local. But multi
 
 > aside positive
 > 
-> **Note:** If you just want to reach the Thread end device from the Wi-Fi/Ethernet network, you can skip this step and go to the "Ping the End Device" step.
+> Note: If you just want to reach the Thread end device from the Wi-Fi/Ethernet network, you can skip this step and go to the "Ping the End Device" step.
```

Comment:
Can you add the bold back where you removed them on the notes?  We have them bold so they look the same as callouts on articles.

Reply:
OK. I can do a sweep where we are inconsistent elsewhere. Example:
https://github.com/openthread/ot-docs/blob/8211ed73ea2479417b94a2d61278bd98d4a6c64f/site/en/codelabs/openthread-border-router-ipv6-multicast/index.lab.md?plain=1#L131